app/Cart.php

```diff
@@ -70,10 +70,10 @@ public function removeItem($item)
     public function updateQuantity($item, $qty)
     {
         // Count the number of $items in array
-        $filteredItems = array_filter($this->items, function($value) use ($item) { return $value === $item; });
```

Comment:
@AC-TimRourke 
I've removed the var `$filteredItems` as it's not needed at this time. Unit test passed so that's okay for now, right?